target/linux/ath79/dts/ar9344_qxwlan_e750a-v4-16m.dts

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344_qxwlan_e750x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Qxwlan E750A V4 16M";
+	compatible = "qxwlan,e750a-v4-16m", "qca,ar9344";
+};
+
+&leds {
+	lan {
+		label = "green:lan";
+		gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+	};
+
+	wan {
+		label = "green:wan";
+		gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+	};
+};
+
+&eth1 {
+	status = "okay";
+
+	mtd-mac-address = <&pridata 0x400>;
+
+	gmac-config {
+		device = <&gmac>;
+		switch-phy-swap = <0>;
+		switch-only-mode = <1>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	phy-handle = <&swphy4>;
+
+	mtd-mac-address = <&pridata 0x400>;
+	mtd-mac-address-increment = <1>;
+};
```

Comment:
I'd prefer to have eth0 node before eth1 here (just stupidly move them).

This actually helps to prevent mistakes when people copy-paste stuff later.

Please do the same for the other files if relevant.